internal/coroutine/coro.go

```diff
@@ -80,7 +75,9 @@ func (p *Coroutines) CreateAndStart(start bool, tobj ThreadObj, fn func(me Threa
 			p.mutex.Unlock()
 			p.sema.Unlock()
 			if e := recover(); e != nil {
-				if e != ErrAbortThread {
+				if pe, ok := e.(iGopPanicError); ok {
+					panic(pe.Error() + "\n" + string(pe.Stack()))
```

Comment:
why not to format error info in outer side?

Reply:
The issue isn't about formatting error info. When user code panics, the panic/recover mechanism here catches and re-throws it, which creates a new stack trace starting from the re-panic point. This makes it impossible to trace back to the original panic location in user code.